swedish translation

Painkiller56 opened the pull request.

Painkiller56 committed:
Added swedich language

Painkiller56 committed:
Added swedich language

Painkiller56 committed:
Updated swedish lang

Looks bad as you dont support international languages (åäö).

Painkiller56 committed:
Swedish translation

Painkiller56 committed:
swedish.lang

Painkiller56 committed:
Update sv.po

Painkiller56 committed:
swedish lang

Painkiller56 committed:
Swedish lang

ä ö changed to umlaut

3d-gussner committed:
Delete ProjectSettings.json

Please only commit `lang/po/new/se.po` files

3d-gussner committed:
Delete .vs/ VSWorkspaceState.json

Please only commit `lang/po/new/se.po` files

3d-gussner reviewed (changes requested):
Please remove the `lang/po/new/.vs/*.*` files from this PR

Please only commit `lang/po/new/se.po` file.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_IR_03_OR_OLDER c=18
+#: messages.c:164
+msgid " 0.3 or older"
+msgstr " 0.3 el \xe1ldre"
```
We replace the `äöüß` and other diacritics with the `lang-import.sh` script.

Please keep the `se.po` file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FS_V_03_OR_OLDER c=18
+#: Marlin_main.cpp:9884
+msgid "FS v0.3 or older"
+msgstr "FS v0.3 el \xe1ldre"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_IR_UNKNOWN c=18
+#: messages.c:165
+msgid "unknown state"
+msgstr "ok\xe1nt tillstand"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MEASURED_OFFSET c=20
+#: ultralcd.cpp:2913
+msgid "[0;0] point offset"
+msgstr "[0;0] punktf\xefrskjutn"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+
+# MSG_CRASH_DET_ONLY_IN_NORMAL c=20 r=4
+msgid "Crash detection can\\x0abe turned on only in\\x0aNormal mode"
+msgstr "Krockdetekt kan\\x0aendast aktiveras i\\x0anormalt l\xe1ge"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+
+# MSG_CRASH_DET_STEALTH_FORCE_OFF c=20 r=4
+msgid "WARNING:\\x0aCrash detection\\x0adisabled in\\x0aStealth mode"
+msgstr "VARNING:\\x0aKrockdetekt\\x0ainaktiverad i\\x0a tyst l\xe1ge"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_DONE c=20 r=8
+#: messages.c:118
+msgid "All is done. Happy printing!"
+msgstr "Allt \xe1r klart. God utskrift!"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CONFIRM_CARRIAGE_AT_THE_TOP c=20 r=2
+#: ultralcd.cpp:3355
+msgid "Are left and right Z~carriages all up?"
+msgstr "\xe1r bada Z-vagnarna helt uppe?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_AUTOLOADING_ONLY_IF_FSENS_ON c=20 r=4
+#: ultralcd.cpp:4317
+msgid "Autoloading filament available only when filament sensor is turned on..."
+msgstr "Autoladdning av fil. \xe1r endast m\xefjligt n\xe1r fil. sensorn \xe1r aktiv...
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+ MSG_AUTOLOADING_ENABLED c=20 r=4
+#: ultralcd.cpp:2648
+msgid "Autoloading filament is active, just press the knob and insert filament..."
+msgstr "Autoladdning filament \xe1r aktiv, tryck pa knappen och s\xe1tt i filament..."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_AXIS_LENGTH c=20
+#: ultralcd.cpp:8173
+msgid "Axis length"
+msgstr "Axell\xe1ngd"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_BEDHEATER c=20
+#: ultralcd.cpp:8131
+msgid "Bed/Heater"
+msgstr "B\xe1dd/V\xe1rmare"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_DONE c=20
+#: messages.c:15
+msgid "Bed done"
+msgstr "B\xe1dd klar"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_HEATING c=20
+#: messages.c:16
+msgid "Bed Heating"
+msgstr "B\xe1dden v\xe1rms upp"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_CORRECTION_MENU c=18
+#: ultralcd.cpp:5798
+msgid "Bed level correct"
+msgstr "B\xe1ddnivakorrigering"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BELTTEST c=18
+#: ultralcd.cpp:5778
+msgid "Belt test"
+msgstr "B\xe1lt test"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_LEVELING_FAILED_POINT_LOW c=20 r=6
+#: messages.c:17
+msgid "Bed leveling failed. Sensor didn't trigger. Debris on nozzle? Waiting for reset."
+msgstr "B\xe1ddnivelleringen felade. Sensorn utl\xefstes inte. Skr\xe1p pa munstycket? V\xe1ntar pa aterst\xe1llning."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED c=13
+#: messages.c:14
+msgid "Bed"
+msgstr "B\xe1dd"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BELT_STATUS c=18
+#: messages.c:19
+msgid "Belt status"
+msgstr "B\xe1lt status"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_RECOVER_PRINT c=20 r=2
+#: messages.c:82
+msgid "Blackout occurred. Recover print?"
+msgstr "Blackout intr\xe1ffat. Aterst\xe1lla utskr?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MOVE_CARRIAGE_TO_THE_TOP c=20 r=8
+#: ultralcd.cpp:3318
+msgid "Calibrating XYZ. Rotate the knob to move the Z carriage up to the end stoppers. Click when done."
+msgstr "Kalibrerar XYZ. Vrid vredet f\xefr att flytta Z-vagnarna upp till \xe1ndstoppen. Klicka n\xe1r du \xe1r klar."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_HOMEYZ_DONE c=20
+#: ultralcd.cpp:656
+msgid "Calibration done"
+msgstr "Kalibraring utf\xefrd"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_NOT_COLOR c=19
+#: ultralcd.cpp:2565
+msgid "Color not correct"
+msgstr "F\xe1rg ej korrekt"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CHOOSE_FIL_1ST_LAYERCAL c=20 r=7
+#: ultralcd.cpp:4842
+msgid "Choose a filament for the First Layer Calibration and select it in the on-screen menu."
+msgstr "V\xe1lj ett filament f\xefr f\xefrsta lagrets kalibrering och v\xe1lj det i sk\xe1rmmenyn."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CRASH_DETECTED c=20
+#: messages.c:29
+msgid "Crash detected."
+msgstr "Krock uppt\xe1ckt."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CRASH_RESUME c=20 r=3
+#: Marlin_main.cpp:651
+msgid "Crash detected. Resume print?"
+msgstr "Krock uppt\xe1ckt. Ateruppta utskrift?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_COMMUNITY_MADE c=18
+#: messages.c:25
+msgid "Community made"
+msgstr "Allm\xe1nhetsgjord"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BABYSTEP_Z_NOT_SET c=20 r=12
+#: messages.c:13
+msgid "Distance between tip of the nozzle and the bed surface has not been set yet. Please follow the manual, chapter First steps, section First layer calibration."
+msgstr "Avstandet mellan munstyckets spets och b\xe1ddytan har inte st\xe1llts in \xe1nnu. V\xe1nligen f\xeflj manualen F\xefrsta lagrets kalibrering."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_REPEAT_V2_CAL c=20 r=7
+#: ultralcd.cpp:5021
+msgid "Do you want to repeat last step to readjust distance between nozzle and heatbed?"
+msgstr "Vill du upprepa det sista steget f\xefr att justera avstandet mellan munstycket och v\xe1rmeb\xe1edden?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_ENDSTOP_NOTHIT c=20
+#: ultralcd.cpp:8149
+msgid "Endstop not hit"
+msgstr "\xe1ndlage inte natt"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_ENDSTOP c=16
+#: ultralcd.cpp:8144
+msgid "Endstop"
+msgstr "\xe1ndl\xe1ge"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_ENDSTOPS c=20
+#: ultralcd.cpp:8135
+msgid "Endstops"
+msgstr "\xe1ndl\xe1gen"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+
+# MSG_STACK_ERROR c=20 r=4
+msgid "Error - static memory has been overwritten"
+msgstr "Fel - statiskt minne har skrivits \xefver"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CUT_FILAMENT c=16
+#: messages.c:61
+msgid "Cut filament"
+msgstr "Sk\xe1r filament"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CUTTER c=9
+#: messages.c:128
+msgid "Cutter"
+msgstr "Sk\xe1rare"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MMU_CUTTING_FIL c=18
+#: mmu.cpp:1387
+msgid "Cutting filament"
+msgstr "Sk\xe1r filament"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_EXTRUDER_FAN_SPEED c=18
+#: ultralcd.cpp:8498
+msgid "Extruder fan:"
+msgstr "Extruderfl\xe1kt:"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FAN_SPEED c=14
+#: messages.c:36
+msgid "Fan speed"
+msgstr "Fl\xe1ktfart"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_FAN c=20
+#: messages.c:91
+msgid "Fan test"
+msgstr "Fl\xe1kttest"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FANS_CHECK c=13
+#: messages.c:33
+msgid "Fans check"
+msgstr "Fl\xe1ktcheck"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FILAMENT_CLEAN c=20 r=2
+#: messages.c:37
+msgid "Filament extruding & with correct color?"
+msgstr "Extruderas filament med r\xe1tt f\xe1rg?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FILE_INCOMPLETE c=20 r=3
+#: ultralcd.cpp:8634
+msgid "File incomplete. Continue anyway?"
+msgstr "Filen \xe1r ofullst\xe1ndig. Forts\xe1tta \xe1nda?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_V2_CALIBRATION c=18
+#: messages.c:124
+msgid "First layer cal."
+msgstr "F\xefrstalager kalib."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_SELFTEST c=20 r=8
+#: ultralcd.cpp:4942
+msgid "First, I will run the selftest to check most common assembly problems."
+msgstr "F\xefrst kommer jag att utf\xefra sj\xe1lvtestet f\xe1r att kontrollera de vanligaste monteringsproblemen."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MMU_FIX_ISSUE c=20 r=4
+#: mmu.cpp:727
+msgid "Fix the issue and then press button on MMU unit."
+msgstr "Atg\xe1rda problemet och tryck sedan pa knappen pa MMU-enheten."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FLOW c=15
+#: ultralcd.cpp:6888
+msgid "Flow"
+msgstr "Fl\xefde"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_COOLING_FAN c=20
+#: messages.c:88
+msgid "Front print fan?"
+msgstr "Frontfl\xe1kt?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_FANS c=20
+#: ultralcd.cpp:8179
+msgid "Front/left fans"
+msgstr "Front/v\xe1nster fl\xe1kt"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_HEATERTHERMISTOR c=20
+#: ultralcd.cpp:8127
+msgid "Heater/Thermistor"
+msgstr "V\xe1rmare/Termistor"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_HEATING_SAFETY_DISABLED c=20 r=4
+#: Marlin_main.cpp:9874
+msgid "Heating disabled by safety timer."
+msgstr "Uppv\xe1rmning avaktiverad av s\xe1kerhetstimer."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_HEATING_COMPLETE c=20
+#: messages.c:51
+msgid "Heating done."
+msgstr "Uppv\xe1rmning klar."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_HEATING c=20
+#: messages.c:50
+msgid "Heating"
+msgstr "Uppv\xe1rmning"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_WELCOME c=20 r=7
+#: messages.c:121
+msgid "Hi, I am your Original Prusa i3 printer. Would you like me to guide you through the setup process?"
+msgstr "Hej, jag \xe1r din Prusa i3-skrivare. Vill du att jag ska guida dig genom installationsprocessen?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FILAMENTCHANGE c=18
+#: messages.c:43
+msgid "Change filament"
+msgstr "\xe1ndra filament"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CHANGE_SUCCESS c=20
+#: ultralcd.cpp:2494
+msgid "Change success!"
+msgstr "\xe1ndring utf\xefrd!"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CORRECTLY c=20
+#: ultralcd.cpp:2562
+msgid "Changed correctly?"
+msgstr "\xe1ndring korrekt?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_CHECK_BED c=20
+#: messages.c:94
+msgid "Checking bed"
+msgstr "Kontroll b\xe1dd"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_CHECK_ENDSTOPS c=20
+#: ultralcd.cpp:8481
+msgid "Checking endstops"
+msgstr "Kontroll \xe1ndl\xe1gen"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CHOOSE_EXTRUDER c=20
+#: messages.c:54
+msgid "Choose extruder:"
+msgstr "V\xe1lj extruder:"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CHOOSE_FILAMENT c=20
+#: messages.c:55
+msgid "Choose filament:"
+msgstr "V\xe1lj filament:"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_XYZ_CAL c=20 r=8
+#: ultralcd.cpp:4951
+msgid "I will run xyz calibration now. It will take approx. 12 mins."
+msgstr "Jag kommer att utf\xefra en xyz-kalibrering nu. Det kommer att ta ca. 12 min."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_Z_CAL c=20 r=8
+#: ultralcd.cpp:4959
+msgid "I will run z calibration now."
+msgstr "Jag kommer att utf\xefra z-kalibrering nu."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WATCH c=18
+#: messages.c:116
+msgid "Info screen"
+msgstr "Infosk\xe1rm"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_INSERT_FILAMENT c=20
+#: ultralcd.cpp:2482
+msgid "Insert filament"
+msgstr "S\xe1tt i filament"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FILAMENT_LOADED c=20 r=2
+#: messages.c:38
+msgid "Is filament loaded?"
+msgstr "\xe1r filament isatt?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_STEEL_SHEET_CHECK c=20 r=2
+#: messages.c:109
+msgid "Is steel sheet on heatbed?"
+msgstr "Ligger metallskiva pa v\xe1rmeb\xe1dden?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_WELCOME_SHIPPING c=20 r=16
+#: messages.c:122
+msgid "Hi, I am your Original Prusa i3 printer. I will guide you through a short setup process, in which the Z-axis will be calibrated. Then, you will be ready to print."
+msgstr "Hej, jag \xe1r din Prusa i3-skrivare. Jag guidar dig genom en kort installationsprocess, d\xe1r Z-axeln kommer att kalibreras. Sedan \xe1r du redo att skriva ut."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_ADDITIONAL_SHEETS c=20 r=9
+#: ultralcd.cpp:5029
+msgid "If you have additional steel sheets, calibrate their presets in Settings - HW Setup - Steel sheets."
+msgstr "Om du har ytterligare metallskivor, kalibrera deras f\xefrinst\xe1llningar i Inst\xe1llningar - HW inst\xe1llning - Metallskiva."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_EXTRUDER_FAN c=20
+#: messages.c:89
+msgid "Left hotend fan?"
+msgstr "V\xe1nst hotend fl\xe1kt?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_LEFT c=10
+#: ultralcd.cpp:2844
+msgid "Left"
+msgstr "V\xe1nster"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_CORRECTION_LEFT c=14
+#: ultralcd.cpp:3114
+msgid "Left side [um]"
+msgstr "V\xe1nstsida [um]"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_LIN_CORRECTION c=18
+#: ultralcd.cpp:5702
+msgid "Lin. correction"
+msgstr "Linj\xe1r korrektion"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_LOADING_COLOR c=20
+#: ultralcd.cpp:2516
+msgid "Loading color"
+msgstr "Laddar f\xe1rg"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_M117_V2_CALIBRATION c=25
+#: messages.c:62
+msgid "M117 First layer cal."
+msgstr "M117 F\xefrst lager kalibr."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BL_LOW c=12
+#: messages.c:156
+msgid "Level Dimmed"
+msgstr "Niva d\xe1mpad"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MEASURE_BED_REFERENCE_HEIGHT_LINE1 c=20 r=3
+#: messages.c:67
+msgid "Measuring reference height of calibration point"
+msgstr "M\xe1tning av referensh\xefjd f\xefr kalibreringspunkt"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MESH_BED_LEVELING c=18
+#: messages.c:148
+msgid "Mesh Bed Leveling"
+msgstr "B\xe1dd nivautj\xe1mning"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MEASURED_SKEW c=14
+#: ultralcd.cpp:2885
+msgid "Measured skew"
+msgstr "M\xe1tt skevhet"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MK3_FIRMWARE_ON_MK3S c=20 r=4
+#: Marlin_main.cpp:882
+msgid "MK3 firmware detected on MK3S printer"
+msgstr "MK3-firmware uppt\xe1ckt pa MK3S-skrivare"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MMU_USER_ATTENTION c=20 r=3
+#: mmu.cpp:722
+msgid "MMU needs user attention."
+msgstr "MMU fordrar anv\xe1ndarens uppm\xe1rksamhet.."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MMU_POWER_FAILS c=15
+#: ultralcd.cpp:1517
+msgid "MMU power fails"
+msgstr "MMU str\xefmavbr"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_HIGH_POWER c=10
+#: messages.c:104
+msgid "High power"
+msgstr "H\xefg kraft"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_NO_MOVE c=20
+#: Marlin_main.cpp:5852
+msgid "No move."
+msgstr "Ingen r\xefrelse."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_NEW_FIRMWARE_AVAILABLE c=20 r=2
+#: util.cpp:195
+msgid "New firmware version available:"
+msgstr "Ny firmware version tillg\xe1nglig:"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_NA c=3
+#: messages.c:127
+msgid "N/A"
+msgstr "N/A"
```
If translation is the same as origin, please leave it empty to safe some space

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_V2_CAL c=20 r=8
+#: ultralcd.cpp:4838
+msgid "Now I will calibrate distance between tip of the nozzle and heatbed surface."
+msgstr "Nu ska jag kalibrera avstandet mellan munstyckets spets och v\xe1rmeb\xe1ddsytan."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_WILL_PREHEAT c=20 r=4
+#: ultralcd.cpp:4967
+msgid "Now I will preheat nozzle for PLA."
+msgstr "Nu ska jag f\xefrv\xe1rma munstycket f\xefr PLA."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_DEFAULT_SETTINGS_LOADED c=20 r=6
+#: Marlin_main.cpp:1605
+msgid "Old settings found. Default PID, Esteps etc. will be set."
+msgstr "Gamla inst\xe1llningar hittades. Standard PID, Esteps etc. kommer att st\xe1llas in."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_NOZZLE_FAN c=10
+#: ultralcd.cpp:1446
+msgid "Nozzle FAN"
+msgstr "Muns fl\xe1kt"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PINDA_PREHEAT c=20
+#: ultralcd.cpp:683
+msgid "PINDA Heating"
+msgstr "PINDA uppv\xe1rmning"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PAPER c=20 r=10
+#: messages.c:73
+msgid "Place a sheet of paper under the nozzle during the calibration of first 4 points. If the nozzle catches the paper, power off the printer immediately."
+msgstr "Placera ett pappersark under munstycket under kalibreringen av de f\xefrsta 4 punkterna. St\xe1ng av skrivaren omedelbart om munstycket r\xefr vid pappret.."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_CLEAN_HEATBED c=20 r=8
+#: ultralcd.cpp:5024
+msgid "Please clean heatbed and then press the knob."
+msgstr "Reng\xefr b\xe1dden och tryck sedan pa knappen."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CONFIRM_NOZZLE_CLEAN c=20 r=8
+#: messages.c:26
+msgid "Please clean the nozzle for calibration. Click when done."
+msgstr "Reng\xefr munstycket f\xefr kalibrering. Klicka n\xe1r du \xe1r klar."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_CALIBRATION_FAILED c=20 r=8
+#: messages.c:117
+msgid "Please check our handbook and fix the problem. Then resume the Wizard by rebooting the printer."
+msgstr "Se var handbok och atg\xe1rda problemet. Ateruppta sedan guiden genom att starta om skrivaren."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CHECK_IDLER c=20 r=5
+#: Marlin_main.cpp:3798
+msgid "Please open idler and remove filament manually."
+msgstr "\xefppna idler och ta bort filamentet manuellt."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PLACE_STEEL_SHEET c=20 r=5
+#: messages.c:75
+msgid "Please place steel sheet on heatbed."
+msgstr "Placera metallskiva pa v\xe1rmeb\xe1dden."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PRESS_TO_UNLOAD c=20 r=4
+#: messages.c:79
+msgid "Please press the knob to unload filament"
+msgstr "V\xe1nligen tryck pa knappen f\xefr att mata ut filament"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PULL_OUT_FILAMENT c=20 r=4
+#: messages.c:81
+msgid "Please pull out filament immediately"
+msgstr "V\xe1nligen ta ut filamentet omedelbart"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_REMOVE_STEEL_SHEET c=20 r=4
+#: messages.c:84
+msgid "Please remove steel sheet from heatbed."
+msgstr "Ta bort metallskivan fran v\xe1rmeb\xe1dden."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_RUN_XYZ c=20 r=4
+#: Marlin_main.cpp:5338
+msgid "Please run XYZ calibration first."
+msgstr "Utf\xefr XYZ-kalibrering f\xefrst."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PLEASE_WAIT c=20
+#: messages.c:76
+msgid "Please wait"
+msgstr "V\xe1nligen v\xe1nta"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_REMOVE_SHIPPING_HELPERS c=20 r=3
+#: ultralcd.cpp:4957
+msgid "Please remove shipping helpers first."
+msgstr "V\xe1nligen ta bort fraktinsatserna f\xefrst."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PREHEAT_NOZZLE c=20
+#: messages.c:78
+msgid "Preheat the nozzle!"
+msgstr "F\xefrv\xe1rm munstycket!"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PREHEAT c=18
+#: ultralcd.cpp:6655
+msgid "Preheat"
+msgstr "F\xefrv\xe1rm"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_HEATING c=20 r=3
+#: messages.c:119
+msgid "Preheating nozzle. Please ewait."
+msgstr "F\xefrv\xe1rmer munstycke. V\xe1nta."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_NEW_FIRMWARE_PLEASE_UPGRADE c=20
+#: util.cpp:199
+msgid "Please upgrade."
+msgstr "V\xe1nligen uppgradera."
```
v

We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PRESS_TO_PREHEAT c=20 r=4
+#: Marlin_main.cpp:12049
+msgid "Press the knob to preheat nozzle and continue."
+msgstr "Tryck pa knappen f\xefr att f\xefrv\xe1rma munstycket och forts\xe1tta."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_POWER_FAILURES c=15\xe1\xe1zzz
+#: messages.c:77
+msgid "Power failures"
+msgstr "Str\xefmavbrott"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PRINT_ABORTED c=20
+#: messages.c:80
+msgid "Print aborted"
+msgstr "Utskriften avbr\xefts"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PREHEATING_TO_LOAD c=20
+#: ultralcd.cpp:2297
+msgid "Preheating to load"
+msgstr "F\xefrv\xe1rmer f\xefr laddn"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+#MSG_PREHEATING_TO_UNLOAD c=20
+#: ultralcd.cpp:2302
+msgid "Preheating to unload"
+msgstr "F\xefrv\xe1rmer for utmatn"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_PRINT_FAN_SPEED c=18
+#: ultralcd.cpp:8501
+msgid "Print fan:"
+msgstr "Utskriftsfl\xe1kt:"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_RESUME_NOZZLE_TEMP c=20 r=4
+#: mmu.cpp:726
+msgid "Press the knob to resume nozzle temperature."
+msgstr "Tryck pa knappen f\xefr att aterst\xe1lla munstyckstemperaturen."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FOLLOW_CALIBRATION_FLOW c=20 r=8
+#: messages.c:46
+msgid "Printer has not been calibrated yet. Please follow the manual, chapter First steps, section Calibration flow."
+msgstr "Skrivaren har inte kalibrerats \xe1nnu. V\xe1nligen f\xeflj manualen, kapitel F\xefrsta stegen, avsnitt Kalibreringsfl\xefde."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PRINT_FAN c=10
+#: ultralcd.cpp:1447
+msgid "Print FAN"
+msgstr "Utskrfl\xe1kt"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_LOAD_FILAMENT c=20 r=6
+#: ultralcd.cpp:4818
+msgid "Please insert filament into the extruder, then press the knob to load it."
+msgstr "V\xe1nligen s\xe1tt i filamentet i extrudern, tryck sedan pa knappen f\xefr inladdning.."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MMU_INSERT_FILAMENT_FIRST_TUBE c=20 r=6
+#: ultralcd.cpp:4813
+msgid "Please insert filament into the first tube of the MMU, then press the knob to load it."
+msgstr "V\xe1nligen s\xe1tt i filamentet i den f\xefrsta kanalen pa MMU:n och tryck sedan pa knappen f\xefr inladdning."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PLEASE_LOAD_PLA c=20 r=4
+#: ultralcd.cpp:4735
+msgid "Please load filament first."
+msgstr "V\xe1nligen ladda filament f\xefrst."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_UNLOAD_FILAMENT_REPEAT c=20 r=4
+#: ultralcd.cpp:7404
+msgid "Please unload the filament first, then repeat this action."
+msgstr "V\xe1nligen mata ut filamentet f\xefrst och upprepa sedan denna atg\xe1rd."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_REMOVE_OLD_FILAMENT c=20 r=5
+#: mmu.cpp:833
+msgid "Remove old filament and press the knob to start loading new filament."
+msgstr "Ta bort det gamla filamentet och tryck pa knappen f\xefr att b\xefrja ladda nytt filament."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CALIBRATE_BED_RESET c=18
+#: ultralcd.cpp:5804
+msgid "Reset XYZ calibr."
+msgstr "Aterst\xe1ll XYZ-kal."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_RESET c=14
+#: messages.c:85
+msgid "Reset"
+msgstr "Aterst\xe1ll"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_CORRECTION_RIGHT c=14
+#: ultralcd.cpp:3115
+msgid "Right side[um]"
+msgstr "H\xefger sida[um]"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_RERUN c=20 r=7
+#: ultralcd.cpp:4756
+msgid "Running Wizard will delete current calibration results and start from the beginning. Continue?"
+msgstr "Att utf\xefra guiden kommer att radera aktuella kalibreringsinst\xe1llningar och b\xefrja om fran b\xefrjan. Forts\xe1tta?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_RIGHT c=10
+#: ultralcd.cpp:2845
+msgid "Right"
+msgstr "H\xefger"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FIND_BED_OFFSET_AND_SKEW_LINE1 c=20 r=3
+#: messages.c:44
+msgid "Searching bed calibration point"
+msgstr "S\xefker efter kalibreringspunkt f\xefr b\xe1dden"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_LANGUAGE_SELECT c=18
+#: ultralcd.cpp:4451
+msgid "Select language"
+msgstr "V\xe1lj sprak"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_OK c=20
+#: ultralcd.cpp:7679
+msgid "Self test OK"
+msgstr "Sj\xe1lvtest OK"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_START c=20
+#: ultralcd.cpp:7447
+msgid "Self test start"
+msgstr "Sj\xe1lvteststart"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST c=18
+#: ultralcd.cpp:5780
+msgid "Selftest"
+msgstr "Sj\xe1lvtest"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_ERROR c=20
+#: ultralcd.cpp:8121
+msgid "Selftest error!"
+msgstr "Sj\xe1lvtestfel!"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_FAILED c=20
+#: messages.c:90
+msgid "Selftest failed"
+msgstr "Sj\xe1lvtestet felade"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FORCE_SELFTEST c=20 r=8
+#: Marlin_main.cpp:1637
+msgid "Selftest will be run to calibrate accurate sensorless rehoming."
+msgstr "Sj\xe1lvtest kommer att utf\xefras f\xefr att kalibrera exakt sensorl\xefs hemposition."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SEL_PREHEAT_TEMP c=20 r=6
+#: ultralcd.cpp:4998
+msgid "Select nozzle preheat temperature which matches your material."
+msgstr "V\xe1lj munstycksf\xefrv\xe1rmningstemperatur som passar ditt material."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SET_TEMPERATURE c=19
+#: ultralcd.cpp:3135
+msgid "Set temperature:"
+msgstr "S\xe1tt temperatur:"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SETTINGS c=18
+#: messages.c:99
+msgid "Settings"
+msgstr "Inst\xe1llningar"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SHOW_END_STOPS c=18
+#: ultralcd.cpp:5801
+msgid "Show end stops"
+msgstr "Visa \xe1ndl\xe1gen"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FILE_CNT c=20 r=6
+#: cardreader.cpp:817
+msgid "Some files will not be sorted. Max. No. of files in 1 folder for sorting is 100."
+msgstr "Vissa filer kommer inte att sorteras. Max. antal filer i 1 mapp f\xefr sortering \xe1r 100."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SEVERE_SKEW c=14
+#: ultralcd.cpp:2888
+msgid "Severe skew"
+msgstr "H\xefg skevhet"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SOUND_LOUD c=7
+#: messages.c:144
+msgid "Loud"
+msgstr "H\xe1gt"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SUPPORT c=18
+#: ultralcd.cpp:6756
+msgid "Support"
+msgstr "Support"
```
If translation is same as origin please leave it blank.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELECT_FILAMENT c=20
+#: ultralcd.cpp:4706
+msgid "Select filament:"
+msgstr "V\xe1lj filament:"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELECT_TEMP_MATCHES_MATERIAL c=20 r=4
+#: ultralcd.cpp:4847
+msgid "Select temperature which matches your material."
+msgstr "V\xe1lj temperatur som passar ditt material."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_TEMP_CALIBRATION_DONE c=20 r=12
+#: messages.c:113
+msgid "Temperature calibration is finished and active. Temp. calibration can be disabled in menu Settings->Temp. cal."
+msgstr "Temperaturkalibreringen \xe1r klar och aktiv. Temp. kalibrering kan inaktiveras i menyn Inst\xe1llningar->Temp. kal."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FOLLOW_Z_CALIBRATION_FLOW c=20 r=9
+#: messages.c:47
+msgid "There is still a need to make Z calibration. Please follow the manual, chapter First steps, section Calibration flow."
+msgstr "Det finns fortfarande ett behov av att g\xefra Z-kalibrering. V\xe1nligen f\xeflj manualen, kapitel F\xefrsta stegen, avsnitt Kalibreringsfl\xefde."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_TUNE c=18
+#: ultralcd.cpp:6653
+msgid "Tune"
+msgstr "St\xe1ll in"
```
Is that correct translation? Fine tune is ...

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_TOTAL c=6
+#: messages.c:100
+msgid "Total"
+msgstr "Total"
```
Correct as above it was written different

pkg2000 replied:
It kind of depends on what it is a total of. In this case it should be blank as it will be the same

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_USED c=19
+#: ultralcd.cpp:5928
+msgid "Used during print"
+msgstr "Anv\xe1nds under utskr"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MENU_VOLTAGES c=15
+#: ultralcd.cpp:2050
+msgid "Voltages"
+msgstr "Sp\xe1nning"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_UNKNOWN c=13
+#: ultralcd.cpp:2005
+msgid "unknown"
+msgstr "ok\xe1nd"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_AUTO c=6
+#: messages.c:160
+msgid "Auto"
+msgstr "Auto"
```
If translation is same as origin leave it blank.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_DIM c=6
+#: messages.c:159
+msgid "Dim"
+msgstr "Dim"
```
If translation is same as origin leave it blank

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_INFO_EXTRUDER c=18
+#: ultralcd.cpp:2040
+msgid "Extruder info"
+msgstr "Extruder info"
```
Same as origin

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_EXTRUDER c=17
+#: messages.c:32
+msgid "Extruder"
+msgstr "Extruder"
```
same as origin

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FSENSOR c=12
+#: messages.c:49
+msgid "Fil. sensor"
+msgstr "Fil. sensor"
```
same as origin

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FILAMENT_SENSOR c=20
+#: messages.c:97
+msgid "Filament sensor"
+msgstr "Filament sensor"
```
same as origin

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_ITERATION c=12
+#: messages.c:53
+msgid "Iteration"
+msgstr "Iteration"
```
same as origin

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MODE c=6
+#: messages.c:103
+msgid "Mode"
+msgstr "MMU Mode"
```
Mode NOT MMU Mode!

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_NORMAL c=7
+#: messages.c:107
+msgid "Normal"
+msgstr "Normal"
```
same as origin

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_MOTOR c=18
+#: messages.c:96
+msgid "Motor"
+msgstr "Motor"
```
same as origin

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_RPI_PORT c=13
+#: messages.c:142
+msgid "RPi port"
+msgstr "RPi port"
```
same as origin

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_TEMP_CAL_WARNING c=20 r=4
+#: Marlin_main.cpp:5351
+msgid "Stable ambient temperature 21-26C is needed a rigid stand is required."
+msgstr "Stabil omgivningstemperatur 21-26C kr\xe1vs samt ett styvt stativ."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_TOTAL_FILAMENT c=19
+#: ultralcd.cpp:2735
+msgid "Total filament"
+msgstr "Totalt filament"
```
Totalt or Total? if same as origin leave it blank

pkg2000 replied:
Totalt is correct

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_USERWAIT c=20
+#: Marlin_main.cpp:4347
+msgid "Wait for user..."
+msgstr "Inv\xe1ntar anv\xe1ndare."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WAITING_TEMP c=20 r=4
+#: ultralcd.cpp:3283
+msgid "Waiting for nozzle and bed cooling"
+msgstr "V\xe1ntar pa munstycks- och b\xe1ddkylning"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WAITING_TEMP_PINDA c=20 r=3
+#: ultralcd.cpp:3244
+msgid "Waiting for PINDA probe cooling"
+msgstr "V\xe1ntar pa PINDA-sondens kylning"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CHANGED_BOTH c=20 r=4
+#: Marlin_main.cpp:1597
+msgid "Warning: both printer type and motherboard type changed."
+msgstr "Varning: bade skrivartyp och moderkortstyp har \xe1ndrats."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CHANGED_MOTHERBOARD c=20 r=4
+#: Marlin_main.cpp:1589
+msgid "Warning: motherboard type changed."
+msgstr "Varning: moderkortstyp \xe1ndrad."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_CHANGED_PRINTER c=20 r=4
+#: Marlin_main.cpp:1593
+msgid "Warning: printer type changed."
+msgstr "Varning: skrivartyp har \xe1ndrats."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_TIMEOUT c=12
+#: messages.c:157
+msgid "Timeout"
+msgstr "Timeout"
```
same as origin

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_SKEW_OFFSET_DETECTION_PERFECT c=20 r=8
+#: ultralcd.cpp:3737
+msgid "XYZ calibration ok. X/Y axes are perpendicular. Congratulations!"
+msgstr "XYZ-kalibrering ok. X/Y-axlarna ar vinkelr\xe1ta. Grattis!"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_SKEW_OFFSET_DETECTION_WARNING_FRONT_BOTH_FAR c=20 r=8
+#: ultralcd.cpp:3721
+msgid "XYZ calibration compromised. Front calibration points not reachable."
+msgstr "XYZ-kalibrering komprometterad. Fr\xe1mre kalibreringspunkter kan inte nas."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_SKEW_OFFSET_DETECTION_WARNING_FRONT_RIGHT_FAR c=20 r=8
+#: ultralcd.cpp:3724
+msgid "XYZ calibration compromised. Right front calibration point not reachable."
+msgstr "XYZ-kalibrering komprometterad. H\xefger fr\xe1mre kalibreringspunkter kan inte nas."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_SKEW_OFFSET_DETECTION_POINT_NOT_FOUND c=20 r=6
+#: ultralcd.cpp:3703
+msgid "XYZ calibration failed. Bed calibration point was not found."
+msgstr "XYZ-kalibrering misslyckades. B\xe1ddkalibreringspunkterna hittades inte."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_SKEW_OFFSET_DETECTION_FAILED_FRONT_BOTH_FAR c=20 r=6
+#: ultralcd.cpp:3709
+msgid "XYZ calibration failed. Front calibration points not reachable."
+msgstr "XYZ-kalibrering misslyckades. Fr\xe1mre kalibreringspunkter kan inte nas."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_BED_SKEW_OFFSET_DETECTION_FAILED_FRONT_RIGHT_FAR c=20 r=6
+#: ultralcd.cpp:3712
+msgid "XYZ calibration failed. Right front calibration point not reachable."
+msgstr "XYZ-kalibrering misslyckades. H\xefger fr\xe1mre kalibreringspunkt kan inte nas."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_WIZARD_V2_CAL_2 c=20 r=12
+#: ultralcd.cpp:4850
+msgid "The printer will start printing a zig-zag line. Rotate the knob until you reach the optimal height. Check the pictures in the handbook (Calibration chapter)."
+msgstr "Skrivaren b\xefrjar skriva ut en sicksacklinje. Vrid vredet tills du nar optimal h\xefjd. Kontrollera med bilderna i handboken (Kalibreringskapitlet)."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FIL_FAILED c=20 r=5
+#: ultralcd.cpp:7415
+msgid "Verification failed, remove the filament and try again."
+msgstr "Verifieringen misslyckades, ta bort filamentet och f\xefrs\xefk igen."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_FALSE_TRIGGERING c=20
+#: ultralcd.cpp:8190
+msgid "False triggering"
+msgstr "Felaktig utl\xefsare"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_HW_SETUP c=18
+#: messages.c:102
+msgid "HW Setup"
+msgstr "HW inst\xe1llning"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MK3S_FIRMWARE_ON_MK3 c=20 r=4
+#: Marlin_main.cpp:875
+msgid "MK3S firmware detected on MK3 printer"
+msgstr "MK3S-firmware uppt\xe1ckt pa MK3-skrivare"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_MODE_CHANGE_IN_PROGRESS c=20 r=3
+#: ultralcd.cpp:4366
+msgid "Mode change in progress..."
+msgstr "L\xe1ges\xe1ndring pagar..."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_GCODE_DIFF_CONTINUE c=20 r=4
+#: util.cpp:414
+msgid "G-code sliced for a different level. Continue?"
+msgstr "G-kod genererad f\xefr en annan niva. Forts\xe1tta?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_GCODE_DIFF_CANCELLED c=20 r=7
+#: util.cpp:420
+msgid "G-code sliced for a different level. Please re-slice the model again. Print cancelled."
+msgstr "G-kod genererad f\xefr en annan niva. V\xe1nligen re-generera modellen igen. Utskriften avbr\xefts."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_GCODE_DIFF_PRINTER_CONTINUE c=20 r=5
+#: messages.c:134
+msgid "G-code sliced for a different printer type. Continue?"
+msgstr "G-kod genererad f\xefr en annan skrivartyp. Forts\xe1tta?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_GCODE_DIFF_PRINTER_CANCELLED c=20 r=8
+#: messages.c:135
+msgid "G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."
+msgstr "G-kod genererad f\xefr en annan skrivartyp. V\xe1nligen re-generera modellen igen. Utskriften avbr\xefts."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_GCODE_NEWER_FIRMWARE_CONTINUE c=20 r=5
+#: util.cpp:381
+msgid "G-code sliced for a newer firmware. Continue?"
+msgstr "G-kod genererad f\xefr en nyare firmware. Forts\xe1tta?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_GCODE_NEWER_FIRMWARE_CANCELLED c=20 r=8
+#: util.cpp:387
+msgid "G-code sliced for a newer firmware. Please update the firmware. Print cancelled."
+msgstr "G-kod genererad f\xefr en nyare firmware. V\xe1nligen uppdatera firmware. Utskriften avbr\xefts."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PREHEATING_TO_CUT c=20
+#: ultralcd.cpp:2309
+msgid "Preheating to cut"
+msgstr "F\xefrv\xe1rmer f\xefr sk\xe1ra"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_PREHEATING_TO_EJECT c=20
+#: ultralcd.cpp:2306
+msgid "Preheating to eject"
+msgstr "F\xefrv\xe1mer f\xefr utmatn"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_NOZZLE_DIFFERS_CONTINUE c=20 r=5
+#: util.cpp:294
+msgid "Printer nozzle diameter differs from the G-code. Continue?"
+msgstr "Skrivarmunstyckets diameter skiljer sig fran G-koden. Forts\xe1tta?"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_NOZZLE_DIFFERS_CANCELLED c=20 r=9
+#: util.cpp:301
+msgid "Printer nozzle diameter differs from the G-code. Please check the value in settings. Print cancelled."
+msgstr "Skrivarmunstyckets diameter skiljer sig fran G-koden. Kontrollera v\xe1rdet i inst\xe1llningarna. Utskriften avbr\xefts."
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELFTEST_FS_LEVEL c=20
+#: ultralcd.cpp:8195
+msgid "%s level expected"
+msgstr "%s niva f\xefrv\xe1ntad"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_RENAME c=18
+#: ultralcd.cpp:6579
+msgid "Rename"
+msgstr "D\xefp om"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SELECT c=18
+#: ultralcd.cpp:6572
+msgid "Select"
+msgstr "V\xe1lj"
```
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.

3d-gussner commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1758 @@
+# MSG_SOUND_BLIND c=7
+#: messages.c:146
+msgid "Assist"
+msgstr "Assist"
```
same as origin

3d-gussner reviewed (changes requested):
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the `se.po` file as you write in Swedish with umlaut and diacritics.

Only if you change `lang/lang_en_se.txt` you need to replace the umlaut and diacritics.

Also if the translation is same as origin please leave it blank to save some space.

3d-gussner committed:
Delete Firmware_sv.pot

Painkiller56 committed:
Swedish translation

Changed back to äö.

Painkiller56 committed:
Merge branch 'MK3_3.11.0' of https://github.com/Painkiller56/Prusa-Firmware into MK3_3.11.0

3d-gussner commented:
@Painkiller56 Thanks for the update. As this PR still contains files that should not be committed and some issues in the `sv.po` file, I will merge your changes to https://github.com/prusa3d/Prusa-Firmware/pull/3349

pkg2000 commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1757 @@
+# MSG_SELFTEST_AXIS c=16
+#: ultralcd.cpp:8174
+msgid "Axis"
+msgstr "Axel"
```
Google translate gives me some weird responses, but as long as it is Axis, like an axis of a coordinate system and not an "axle" like you find on a car, this is correct. Not sure of the swedish here.

Painkiller56 replied:
one "axel", many  "axlar".
Correct swedish for coordinate systems.

pkg2000 replied:
Cool, thank you for confirming. Again, the small differences in Nor. Axle would be Aksel, and Axis would be Akse.

pkg2000 commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1757 @@
+# MSG_BED_CORRECTION_MENU c=18
+#: ultralcd.cpp:5798
+msgid "Bed level correct"
+msgstr "Bäddnivakorrigering"
```
Baddplan? It is level as in flat, not as a height

Painkiller56 replied:
How to describe it in limit of 18 chars?
"Bäddytskorrigering" = 18

pkg2000 commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1757 @@
+# MSG_COMMUNITY_MADE c=18
+#: messages.c:25
+msgid "Community made"
+msgstr "Allmänhetsgjord"
```
I struggled with this one in Norwegian. I think I just left it as it was. Does not Allmänhetsgjord translate more to "the general public-done" not a specific community?

Painkiller56 replied:
I to struggled with it :).
"Sällskapsskapad"
"Community skapad"
I think the second is best ?

pkg2000 replied:
I think the second one is best as well. Community skapad, or just leave it blank to save some space.

pkg2000 commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1757 @@
+# MSG_FINISHING_MOVEMENTS c=20
+#: messages.c:45
+msgid "Finishing movements"
+msgstr "Avslutar flyttning"
```
Avslutar rörelse? Perhaps flyttning is correct.

Painkiller56 replied:
Yes

pkg2000 commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1757 @@
+# MSG_SELFTEST_CHECK_BED c=20
+#: messages.c:94
+msgid "Checking bed"
+msgstr "Kontroll bädd"
```
Kontroll**erar** bädd? It is in the process of checking the bed at this message,

Painkiller56 replied:
Context error. Of cours Kontrollerar

pkg2000 commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1757 @@
+# MSG_SELFTEST_CHECK_ENDSTOPS c=20
+#: ultralcd.cpp:8481
+msgid "Checking endstops"
+msgstr "Kontroll ändlägen"
```
Same as above. Not sure if the time of the verb matters here

Painkiller56 replied:
Same as above

pkg2000 commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1757 @@
+# MSG_LOOSE_PULLEY c=20
+#: ultralcd.cpp:8167
+msgid "Loose pulley"
+msgstr "Lossa pulley"
```
Loosening (lossa) the pulley? It is informing of a pulley that could be loose/not secured. Would it not be lösa or.. In Norwegian it would be løs

Painkiller56 replied:
lös in swedish

pkg2000 replied:
lös pulley then.

pkg2000 commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1757 @@
+# MSG_MAIN c=18
+#: messages.c:63
+msgid "Main"
+msgstr "Huvudsaklig"
```
As in main menu, or just main. Huvudsaklig would be more like Mainly, no? So just "huvud"

Painkiller56 replied:
ok

pkg2000 commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1757 @@
+# MSG_SELFTEST_SWAPPED c=16
+#: ultralcd.cpp:8180
+msgid "Swapped"
+msgstr "Bytt"
```
utbytt?

Painkiller56 replied:
Utbytt better.

pkg2000 commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1757 @@
+# MSG_USERWAIT c=20
+#: Marlin_main.cpp:4347
+msgid "Wait for user..."
+msgstr "Inväntar användare."
```
Väntar på

pkg2000 commented on lang/po/new/sv.po:
```diff
@@ -1,1772 +1,1757 @@
+# MSG_CHECKS c=18
+#: ultralcd.cpp:5641
+msgid "Checks"
+msgstr "Kontrollerar"
```
I think this is about "some checks" and not that it "checks" it now.

Painkiller56 committed:
Swedish translation

Corrections of  last translation

Painkiller56 committed:
Swedish translation of firmware

Conveted "aA" to "åÅ" when necessary. Now the translation is natural swedish.

Deleted all instances when english = swedish.

Painkiller56 committed:
Swedish language

Line 787 = "Main", så translation to "Huvudsaklig" is correct.
BUT the english text should be "Main Menu" and therefore the correct translation is "Huvudmeny" (also only used in that context).

3d-gussner commented:
Closed as it has been implemented.
